publication/src/main/scala/hmda/publication/reports/util/ReportsMetadataLookup.scala

```diff
@@ -0,0 +1,49 @@
+package hmda.publication.reports.util
+
+import com.github.tototoshi.csv.CSVParser.parse
+import hmda.model.ResourceUtils
+import hmda.model.publication.reports.ReportTypeEnum
+import hmda.model.publication.reports.ReportTypeEnum.Disclosure
+import hmda.publication.reports.util.DispositionType.{ DispositionType, ReceivedDisp }
+
+object ReportsMetaDataLookup extends ResourceUtils {
+
+  private val lines = resourceLines("/reports-metadata.txt")
+
+  val values: Map[String, ReportMetaData] = lines.drop(1).map { line =>
+    val data = ReportMetaData.fromString(line)
+    data.reportId -> data
+  }.toMap
+
+}
+
+case class ReportMetaData(
+  reportId: String,
```

Comment:
If this is being stored in a map with the report ID as the key, I think this field might be redundant.

Reply:
Strictly yes, that field isn't needed for report generation. But including `reportId` in the `ReportMetaData` was the cleanest way to get access to it on line 15 of this file, so I could make it the key of the Map. Since it's useful in that case, maybe it's okay to leave it?

Reply:
Hmmmmmm... How about moving this line: 
`val values = parse(line, '\\', ',', '"').getOrElse(List())` 
up to line 14, then passing in the resulting array to the `forString` method and using `values.head` as the key?

Reply:
That would totally work, but that seems like that is a worse separation of concerns. 

I guess I don't see why it's important to make the `ReportMetaData` structure as small as possible. Want to explain your reasoning?

Reply:
Ignore my last comment :). I added a potential fix in 703af60000d7bea2a2eeec2d59eccc5223916859. Let me know what you think.